tailwind.config.js

```diff
@@ -1,52 +1,61 @@
 module.exports = {
   purge: {
-    content: ['./dist/**/*.html'],
-    
+    content: ["./dist/**/*.html"],
+
     // Manually whitelist style classes for PurgeCSS to ignore
     options: {
       whitelist: [],
-    }
-},
+    },
+  },
```

Comment:
Does purge still work @JackB47 or do we need to incorporate this manually?

Reply:
I believe it does yeah, the dist file hasn't churned every single tw class out